src/sentry/static/sentry/app/views/organizationGroupDetails/groupEventDetails/groupEventDetails.tsx

```diff
@@ -132,39 +126,17 @@ class GroupEventDetails extends React.Component<Props, State> {
     const projSlug = project.slug;
     const projectId = project.id;
 
-    this.setState({
-      loading: true,
-      error: false,
-    });
-
     /**
      * Perform below requests in parallel
      */
     const releasesCompletionPromise = api.requestPromise(
       `/projects/${orgSlug}/${projSlug}/releases/completion/`
     );
-
     fetchSentryAppInstallations(api, orgSlug);
     fetchSentryAppComponents(api, orgSlug, projectId);
 
     const releasesCompletion = await releasesCompletionPromise;
-    this.setState({
-      releasesCompletion,
-    });
-
-    try {
-      this.setState({
-        error: false,
-        loading: false,
-      });
-    } catch (err) {
-      // This is an expected error, capture to Sentry so that it is not considered as an unhandled error
-      Sentry.captureException(err);
-      this.setState({
-        error: true,
-        loading: false,
-      });
```

Comment:
:thunking-face: what was this trying to catch?

Reply:
Probably the stuff i deleted in the perf PR...